lib/NeuraDialect/Transforms/MapToAcceleratorPass.cpp

```diff
@@ -206,30 +212,36 @@ struct MapToAcceleratorPass
         llvm::outs() << "[MapToAcceleratorPass] Topologically sorted op: "
                      << *op << "\n";
       }
-      std::vector<std::vector<Operation *>> level_buckets =
-          getOpsInAlapLevels(topologically_sorted_ops, critical_ops);
-      for (int level = 0; level < static_cast<int>(level_buckets.size());
-           ++level) {
-        llvm::outs() << "[MapToAcceleratorPass] ALAP Bucket Level " << level
-                     << ": " << level_buckets[level].size() << " ops\n";
-        for (Operation *op : level_buckets[level]) {
-          llvm::outs() << "  " << *op << "\n";
+
+      std::vector<std::pair<Operation *, int>> sorted_ops_with_levels;
+      if (sort_strategy_stringRef == "topological") {
```

Comment:
Add comment above mentioning only two sorting algorithms for now, and by default is `mixed`.